src/skip-to-content/skip-to-content.scss

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-skip-to-content {
+  @include iui-blur($opacity: 3);
+  @include iui-focus($offset: 2px, $thickness: 2px);
+  @include themed {
+    color: t(iui-color-foreground-accessory);
+  }
+
+  border-radius: $iui-border-radius-round;
+  text-decoration: none;
+  padding: ($iui-baseline * 0.75) $iui-m;
+  position: fixed;
+  text-align: center;
+  left: 50%;
```

Comment:
Currently it looks ok, because our body has top padding applied through demo css (specific only for our html pages). In real scenario, we could expect this being put without any spacing from the top. As a result, it would be just sticked to the top.. Maybe we could think of adding `top` or some `margin`... Not sure.

![image](https://user-images.githubusercontent.com/81580355/153560711-a61b18f2-c777-4fd6-9d0b-1e2282c8adb8.png)

Reply:
Good catch, thanks. Fixed.

Reply:
We usually try to use our spacing variables for that. Could you try applying `$iui-baseline` here? Maybe `2x$iui-baseline` would be alright.

Reply:
Right, thanks.

Reply:
Fixed.